screens/upload.js

```diff
@@ -81,18 +81,18 @@ function UploadScreen({ route, navigation }) {
 
   var length = 0;
   var tagList = [];
-  var threeOrFour = true;
+  var three = true;
```

Comment:
I like that you changed this from threeOrFour, but is there a better name, like maxLables, that generalizes?